# code for the predefined pose
string pose
string label
----
bool success
